demo/README.md

```diff
@@ -0,0 +1,13 @@
+# tasking-api
+
+Install packages
+
+```python
+pip install -r requirements.txt
```

Comment:
I typically commit .python-version and include a virtual environments in a .venv folder. Do projects typically leave that up to the developer to figure out so that it can be flexible? I don't work with many python people so I haven't learned the typical ways of doing things.